src/components/utils/onfidoApi.ts

```diff
@@ -382,3 +383,29 @@ export const sendAnalytics = (
 
   request.send(payload)
 }
+
+export const getSdkConfiguration = (
+  url: string,
+  token: string
+): Promise<SdkConfiguration> =>
+  new Promise((resolve, reject) => {
+    if (!process.env.SDK_SOURCE) {
+      throw new Error('SDK_SOURCE not defined')
+    }
+
+    const sdk_source = process.env.SDK_SOURCE.replaceAll('_', '-')
```

Comment:
We don't need to normalise from `onfido_web_sdk` to `onfido-web-sdk`. Our data pipeline has been living with the former value for ages 😅

Reply:
So we need to update the Configuration Service which expects "onfido-web-sdk". Or I put back the hardcoded value.

Reply:
You should keep the `SDK_SOURCE` env var and update sdk-configuration-service to accept `onfido_web_sdk`

Reply:
👍